diff --git a/pyproject.toml b/pyproject.toml
index e14ee0ca..0c4249c3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,12 +31,12 @@ mswms = "mslib.mswms.mswms:main"
 msui = "mslib.msui.msui:main"
 
 [build-system]
-requires = ["setuptools >= 40.8.0"]
+requires = ["setuptools >= 61.0"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-packages = ["mslib"]
-
 [tool.codespell]
 exclude-file = "codespell-ignored-lines.txt"
 ignore-words-list = [
