diff --git a/mslib/msui/mss_pyui.py b/mslib/msui/mss_pyui.py
index ba80231..7608265 100644
--- a/mslib/msui/mss_pyui.py
+++ b/mslib/msui/mss_pyui.py
@@ -30,6 +30,27 @@
 """
 from __future__ import print_function
 
+# Hack to fix missing PROJ4 env var in root environment
+import os
+import sys
+import setuptools
+
+
+if os.getenv("PROJ_LIB") is None or os.getenv("PROJ_LIB") == "PROJ_LIB":
+    conda_file_dir = setuptools.__file__
+    conda_dir = conda_file_dir.split('lib')[0]
+    proj_lib = os.path.join(os.path.join(conda_dir, 'share'), 'proj')
+    if "win" in sys.platform:
+        proj_lib = os.path.join(os.path.join(conda_dir, 'Library'), 'share')
+    os.environ["PROJ_LIB"] = proj_lib
+    if not os.path.exists(proj_lib):
+        os.makedirs(proj_lib)
+        epsg_file = os.path.join(proj_lib, 'epsg')
+        if not os.path.exists(epsg_file):
+            with open(os.path.join(proj_lib, 'epsg'), 'w') as fid:
+                fid.write("# Placeholder for epsg data")
+
+
 import argparse
 import copy
 import functools
