diff --git a/mslib/mswms/mswms.py b/mslib/mswms/mswms.py
index 06391a2..d890885 100644
--- a/mslib/mswms/mswms.py
+++ b/mslib/mswms/mswms.py
@@ -30,6 +30,17 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
+# Hack to fix missing PROJ4 env var in root environment
+import os
+import setuptools
+
+if os.getenv("PROJ_LIB") is None:
+    conda_file_dir = setuptools.__file__
+    conda_dir = conda_file_dir.split('lib')[0]
+    proj_lib = os.path.join(os.path.join(conda_dir, 'share'), 'proj')
+    os.environ["PROJ_LIB"] = proj_lib
+
+
 import paste.httpserver
 import argparse
 import logging
