diff --git a/setup.py b/setup.py
index 59da681..7cac451 100644
--- setup.py
+++ setup.py
@@ -24,6 +24,14 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 """
+# Hack to fix missing PROJ4 env var
+import os
+import setuptools
+
+conda_file_dir = setuptools.__file__
+conda_dir = conda_file_dir.split('lib')[0]
+proj_lib = os.path.join(os.path.join(conda_dir, 'share'), 'proj')
+os.environ["PROJ_LIB"] = proj_lib
 
 # The README.txt file should be written in reST so that PyPI can use
 # it to generate your project's PyPI page.
